src/Native/Runtime/PalRedhawkCommon.h

```diff
@@ -144,7 +150,7 @@ typedef void (__fastcall * ThreadExitCallback)();
 
 extern ThreadExitCallback g_threadExitCallback;
 
-typedef Int32 (*PHARDWARE_EXCEPTION_HANDLER)(UIntNative faultCode, UIntNative faultAddress, PAL_LIMITED_CONTEXT* palContext, UIntNative* arg0Reg, UIntNative* arg1Reg);
+typedef Int32 (__stdcall *PHARDWARE_EXCEPTION_HANDLER)(UIntNative faultCode, UIntNative faultAddress, PAL_LIMITED_CONTEXT* palContext, UIntNative* arg0Reg, UIntNative* arg1Reg);
```

Comment:
It would be better to remove `__stdcall` from where this is used instead. The fewer places with the custom calling convention mods, the better.

Reply:
It is used in startup.cpp, but the actual issue is `PalRedhawh.h` has REDHAWK_PALIMPORT in its definition. Should I instead remove `REDHAWK_PALIMPORT` from the header?

Reply:
I think removing it here https://github.com/dotnet/corert/blob/master/src/Native/Runtime/startup.cpp#L37 should be sufficient.